src/evergreen/evg_config_utils.rs

```diff
@@ -654,6 +661,42 @@ fn get_generate_resmoke_func(task: &EvgTask) -> Option<&FunctionCall> {
     }
 }
 
+/// Get the vars passed to "generate resmoke task" or "run tests".
+///
+/// # Arguments
+///
+/// * `task` - Shrub task to query.
+///
+/// # Returns
+///
+/// vars forwarded to resmoke.py.
+fn get_resmoke_vars(task: &EvgTask) -> Option<&HashMap<String, ParamValue>> {
+    let command = if let Some(commands) = &task.commands {
+        commands.iter().find(|c| {
+            if let Function(func) = c {
+                if func.func == GENERATE_RESMOKE_TASKS {
+                    return true;
+                } else if func.func == RUN_RESMOKE_TASK {
+                    return true;
+                }
+            }
+            false
+        })
+    } else {
+        None
+    };
+
+    if let Some(Function(func)) = command {
+        if let Some(vars) = &func.vars {
+            return Some(vars)
+        } else {
+            None
+        }
+    } else {
+        None
+    }
+}
```

Comment:
And some tests for this as well

Reply:
For this function, my thought was to have it be internal to this module. Since we don't need to mock out functions as much in this project (and rely instead more heavily on traits), the 2 tests added for `find_suite_name` should catch any visible changes to this function. Please feel free to push back

Reply:
That's fine with me